src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -59,7 +59,7 @@ export class NotebookMarkdownRenderer {
 		let signalInnerHTML: () => void;
 		const withInnerHTML = new Promise(c => signalInnerHTML = c);
 
-		let notebookFolder = path.dirname(this._notebookURI.fsPath) + '/';
+		let notebookFolder = this._notebookURI ? path.dirname(this._notebookURI.fsPath) + path.sep : '';
```

Comment:
let's do path.join() here

Reply:
path.join(path.dirname(this._notebookURI.fsPath), path.sep) ?

Reply:
looks right, yup!